controllers/managedocs_controller_test.go

```diff
@@ -973,6 +964,73 @@ var _ = Describe("ManagedOCS controller", func() {
 					utils.WaitForResource(k8sClient, ctx, alertRelabelConfigSecretTemplate.DeepCopy(), timeout, interval)
 				})
 			})
+			When("the kube-rbac-proxy ConfigMap is modified", func() {
+				It("should revert the changes and bring the resource back to its managed state", func() {
+					// Get an updated configMap
+					kubeRBACConfigMap := prometheusKubeRBACPoxyConfigMapTemplate.DeepCopy()
+					kubeRBACConfigMapKey := utils.GetResourceKey(kubeRBACConfigMap)
+					Expect(k8sClient.Get(ctx, kubeRBACConfigMapKey, kubeRBACConfigMap)).Should(Succeed())
+
+					// Update to the configMap with dummy values
+					data := kubeRBACConfigMap.Data
+					kubeRBACConfigMap.Data = map[string]string{
+						"test-key": "test-value",
+					}
```

Comment:
No need to add mock data, we just want to verify that it is reconciling back. An empty object will work just fine
```suggestion
					kubeRBACConfigMap.Data = map[string]string{}
```